modules/core/src/main/java/org/locationtech/jts/algorithm/Centroid.java

```diff
@@ -153,7 +153,6 @@ else if (ptCount > 0){
 
   private void setBasePoint(Coordinate basePt)
```

Comment:
How about we rename this to setAreaBasePoint?  Makes it a teeny bit more descriptive, and indicates that this functionality is only necessary for areas.